cli/azd/pkg/apphost/generate.go

```diff
@@ -880,7 +897,11 @@ func (b infraGenerator) evalBindingRef(v string, emitType inputEmitType) (string
 	case targetType == "postgres.database.v0" ||
 		targetType == "redis.v0" ||
 		targetType == "azure.cosmosdb.account.v0" ||
-		targetType == "azure.cosmosdb.database.v0":
+		targetType == "azure.cosmosdb.database.v0" ||
+		targetType == "azure.sql.v0" ||
+		targetType == "sqlserver.server.v0" ||
```

Comment:
NIT: Please swap L902 and L903 so the two azure. ones are next to each other in the list.  Makes the code a little easier to scan.